cmd/rook/cmdreporter/cmdreporter.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmdreporter
```

Comment:
What about adding cmdreporter to the `k8sutil` package instead of creating a new package? It seems to fit in that package.

Reply:
I would agree, but I'm not sure what you're envisioning. Would you think I would add these command definitions to k8sutil as well?

Reply:
I see now that you have the cmdreporter under `cmd` and `daemon`, which makes sense. I was thinking of the operator side of this, which would need to launch the cmdreporter job and return the output. That's the piece I would envision in the k8sutil package.

Reply:
I'd like to have something like this in the k8sutil package, but the job still has to be somewhat flexible to the calling operator. The job's pod template may need user modifications, for example. Currently, I'm not sure the best way to abstract this, and I'd propose waiting until we have a few examples of the job being called so that we can see what patterns emerge first.